src/components/policy/policy_external/src/policy_manager_impl.cc

```diff
@@ -280,6 +280,51 @@ std::string PolicyManagerImpl::GetLockScreenIconUrl() const {
   return cache_->GetLockScreenIconUrl();
 }
 
+/**
+ * @brief FilterInvalidFunctions filter functions that are not present in schema
```

Comment:
@LuxoftAKutsan Make it shorter: `functions that are not present in schema` -> `functions that are absent in schema`

Reply:
@AByzhynar fixed 829bec8